src/components/ui/filters/Filters.vue

```diff
@@ -0,0 +1,160 @@
+              />
+              <vuestic-simple-select
+                class="filters-page__filter-bar"
+                label="city"
+                v-model="city"
+                :options="cityList"
+              />
+            </div>
+          </div>
+          <div class="filters-page__tags">
+            <vuestic-tag
+              v-if="name"
+              :name="`Name: ${ name }`"
+              removable
+              @remove="name = ''"
+            />
+            <vuestic-tag
+              v-if="email"
+              :name="`Email: ${ email }`"
+              removable
+              @remove="email = ''"
+            />
+            <vuestic-tag
+              v-if="city"
+              :name="`City: ${ city }`"
+              removable
+              @remove="city = ''"
+            />
+            <span
+              v-if="this.city || this.name || this.email"
+              class="filters-page__clear-all-text"
+              @click="clearAll"
+            >
+              Clear all filters
+            </span>
+          </div>
+        </div>
+      </div>
+    </vuestic-widget>
+    <vuestic-widget>
+      <div class="table-responsive">
+        <table class="table table-striped first-td-padding">
+          <thead>
+          <tr>
+            <td class="filters-page__table-heading">{{ $t('tables.headings.name') }}</td>
+            <td class="filters-page__table-heading">{{ $t('tables.headings.email') }}</td>
+            <td class="filters-page__table-heading">{{ $t('tables.headings.city') }}</td>
+            <td align="right" class="filters-page__table-heading">{{ $t('tables.headings.score') }}</td>
+            <td></td>
+          </tr>
+          </thead>
+          <tbody>
+          <tr v-for="item in filteredItems" v-bind:key="item.email">
+            <td>{{ item.name }}</td>
+            <td>{{ item.email }}</td>
+            <td>{{ item.city }}</td>
+            <td align="right">{{ item.score }}</td>
+            <td></td>
+          </tr>
+          </tbody>
+        </table>
+      </div>
+    </vuestic-widget>
+  </div>
+</template>
+
+<script>
+import VuesticWidget
+  from '../../../vuestic-theme/vuestic-components/vuestic-widget/VuesticWidget'
+import FilterBar
+  from '../../../vuestic-theme/vuestic-components/vuestic-datatable/datatable-components/FilterBar.vue'
+import VuesticSimpleSelect
+  from '../../../vuestic-theme/vuestic-components/vuestic-simple-select/VuesticSimpleSelect'
+import { SpringSpinner } from 'epic-spinners'
+import { cityList, itemList } from './filtersData'
+export default {
+  name: 'filters',
+  components: {
+    VuesticWidget, FilterBar, SpringSpinner, VuesticSimpleSelect
+  },
+  data () {
+    return {
+      name: '',
+      email: '',
+      city: '',
+      cityList: cityList,
+      itemList: itemList,
+    }
+  },
+  methods: {
+    clearAll () {
+      this.name = ''
+      this.email = ''
+      this.city = ''
+    }
+  },
+  computed: {
+    filteredItems () {
```

Comment:
Lot's of code duplication, can be somewhat simplified:
```
filteredItems () {
  return this.itemList
    .filter(item => this.hasMatch(item.name, this.name))
    .filter(item => this.hasMatch(item.email, this.email))
    .filter(item => this.hasMatch(item.city, this.city))
}
```

cc @asvae

Reply:
@jean-moldovan 
I agree on name and email.
For entities search could be more complex, so it's principal to be explicit.

Reply:
ok